data/person/01K5ST4FV2EHYZ7XATD4C0DSV6.toml

```diff
@@ -3,19 +3,11 @@ congress_website_primary_keys = [ 292,]
 congress_website_author_keys = [ "J078",]
 last_name = "Valeriano"
 first_name = "Rolando"
-middle_name = "M"
+middle_name = "Mayodato"
+aliases = ["Rolan", "Rolan CRV",]
 
 [[memberships]]
 type = "congress"
 congress = 18
 position = "representative"
 
-[[memberships]]
```

Comment:
Should not be removed:

https://en.wikipedia.org/wiki/19th_Congress_of_the_Philippines
https://en.wikipedia.org/wiki/20th_Congress_of_the_Philippines